app/views/orgs/lti_configurations/show.html.erb

```diff
@@ -1,25 +1,27 @@
 <div class="remodal-bg">
   <%= render 'organizations/organization_banner' %>
 
+  <% lms_name = current_lti_configuration.lms_name(default_name: "your Learning Management System") %>
+
   <div class="columns">
     <div class="one-fourth column">
       <%= render partial: 'organizations/side_menu' %>
     </div>
     <div class="three-fourths column">
       <div class="site-content">
         <div class="site-content-cap">
-          <h2 class="site-content-heading">Learning Management System Configuration</h2>
+          <h2 class="site-content-heading">Connect to <%= lms_name %></h2>
```

Comment:
This will say `Connect to Other Learning Management System` if they're using a non top-5 LMS. This reads strangely, shouldn't it be something more like `Connect to Learning Management System`? If a user was using one of these other systems, it'll be referred to as a `Other` system throughout the app, forever.

Reply:
Yeah, so here we’re actually using the feature for `lms_name` that allows us to specify a different default depending on context. Here specifically it would say, “Connect to your Learning Management System”